File: 1.Two Sum/solution.rb


def two_number(number_list,target)
    number_list.each_with_index do |num,i|
        missing = target - num
        missing_index = number_list.index(missing)
        if missing_index != i
            puts "#{i} #{missing_index}"
        end
    end
    
end


two_number([3,2,4],6)

File: 1.Two Sum/solution.py


def two_number(number_list,target):
    for i,num in enumerate(number_list):
        missing = target - num
        if missing in number_list and missing != num:
            missing_index = number_list.index(missing)
            print(i,missing_index)

two_number([3,2,4],6)